packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -40,56 +41,193 @@ const LOADPANEL_MESSAGE_CLASS = 'dx-loadpanel-message';
 const MESSAGE_SELECTOR = '.' + LOADPANEL_MESSAGE_CLASS;
 const LOADPANEL_CONTENT_CLASS = 'dx-loadpanel-content';
 const LOADPANEL_PANE_HIDDEN_CLASS = 'dx-loadpanel-pane-hidden';
+const LOADINDICATOR_CLASS = '.dx-loadpanel-indicator';
+
+QUnit.module('accessibility', () => {
+    QUnit.module('message option tests', () => {
+        ['', undefined, null].forEach(state => {
+            const stateString = (state === '') ? 'empty' : state;
+            QUnit.test(`message is set to ${stateString} on init`, function(assert) {
+                const loadPanel = $('#loadPanel').dxLoadPanel({
+                    visible: true,
+                    message: state,
+                    showIndicator: false
+                }).dxLoadPanel('instance');
+                const loadPanelMessage = loadPanel.$wrapper().find('.dx-loadpanel-message').text();
 
-QUnit.module('init', {
-    beforeEach: function() {
-        fx.off = true;
-    },
+                assert.strictEqual(loadPanelMessage, '', 'Load Panel message value is empty');
+            });
 
-    afterEach: function() {
-        fx.off = false;
-    }
-}, () => {
-    QUnit.test('rendered markup', function(assert) {
-        const $element = $('#loadPanel').dxLoadPanel({ message: 'Test Loading Message', visible: true }); const $content = $element.dxLoadPanel('instance').$content();
+            QUnit.test(`message is set from ${stateString} to non empty on runtime`, function(assert) {
+                const loadPanel = $('#loadPanel').dxLoadPanel({
+                    visible: true,
+                    message: state,
+                    showIndicator: false
+                }).dxLoadPanel('instance');
+                let loadPanelMessage = loadPanel.$wrapper().find(`.${LOADPANEL_MESSAGE_CLASS}`).text();
 
-        assert.ok($element.hasClass(LOADPANEL_CLASS));
-        assert.ok($content.hasClass(LOADPANEL_CONTENT_CLASS), 'Load Indicator created');
-        assert.ok($content.find(MESSAGE_SELECTOR).length);
-        assert.equal($content.find(MESSAGE_SELECTOR).text(), 'Test Loading Message');
+                assert.strictEqual(loadPanelMessage, '', 'Load Panel message value is empty');
+
+                assert.timeout(1000);
```

Comment:
For what so you use this timeouts? For animation skipping? Than it's better just disabled the animation in before each how it's don't in a neighbor module below.

Reply:
[bb57796](https://github.com/DevExpress/DevExtreme/pull/27080/commits/bb57796b0f4e0953f491b0adee68e1f35762a715)